gamehub-backend/src/main/java/com/example/gamehubbackend/controllers/GameController.java

```diff
@@ -69,10 +68,12 @@ public Game updateGame(@PathVariable String id, @RequestBody GameDTO game) {
      * Delete a game from the database.
      *
      * @param id the ID of the game to delete
+     * @return no content and status 204
      */
     @DeleteMapping("/{id}")
-    public void deleteGame(@PathVariable String id) {
+    public ResponseEntity<Object> deleteGame(@PathVariable String id) {
```

Comment:
is there a benefit of returning this instead of just giving the `@ResponseStatus(HttpStatus.NO_CONTENT)` annotation at the top of the method?

Reply:
no  :D just liked it that way, because if you are using jpa you do it this way too.